### PR TITLE
feat(kubernetes-v2): first steps for a v2 deployer

### PR DESCRIPTION
**Pull request by lwander**
Remaining work is listed here: https://github.com/spinnaker/spinnaker/issues/2176

### CODE REVIEW

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/services/v1/ResponseUnwrapper.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
 public class ResponseUnwrapper {
   private static final Long WAIT_MILLIS = 400L;
   private static int cycle;
-  private static String[] cursors = {".", "-", "^"};
+  private static String[] cursors = {"_", "-", "^", "*"};
```
Comment (jtk54) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/services/v1/ResponseUnwrapper.java]: Nice.

File: halyard-core/halyard-core.gradle [hunk 2]
```diff
@@ -6,4 +6,5 @@ dependencies {
   compile spinnaker.dependency('googleStorage')
 
   compile 'com.beust:jcommander:1.48'
+  compile 'com.hubspot.jinjava:jinjava:2.2.3'
```
Comment (jtk54) [inline halyard-core/halyard-core.gradle]: This exists in a few places now, need to add to spinnaker-dependencies eventually.

File: halyard-core/src/main/java/com/netflix/spinnaker/halyard/core/resource/v1/JarResourceReader.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Google, Inc.
+ * Copyright 2018 Google, Inc.
```
Comment (jtk54) [inline halyard-core/src/main/java/com/netflix/spinnaker/halyard/core/resource/v1/JarResourceReader.java]: I thought this was representing file creation?

Comment (lwander) [reply halyard-core/src/main/java/com/netflix/spinnaker/halyard/core/resource/v1/JarResourceReader.java]: Good catch

File: halyard-core/halyard-core.gradle (hunk 2 above)
Comment (lwander) [reply halyard-core/halyard-core.gradle]: It's still pretty fresh in our codebase - both the artifact & (now this) is largely unproven

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/services/v1/ResponseUnwrapper.java (hunk 1 above)
Comment (lwander) [reply halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/services/v1/ResponseUnwrapper.java]: I think this is a 1.0 worthy change

**Pull request merged**